.gitignore

```diff
@@ -5,3 +5,5 @@ CMakeFiles
 CMakeCache.txt
 .vscode
 build/*
+DataBase/mnist_train.csv
+DataBase/mnist_test.csv
```

Comment:
Is this so when you unzip the data that it doesn't get added to the repository?

Reply:
yeah